### PR TITLE
Add index&stylesheet

### PR DESCRIPTION
**Pull request by AlexHamn**
Add working HTML skeleton & linked CSS file

### CODE REVIEW

**Comment by AlexHamn** [comment]
Hi Alex! :D
I'm so very glad to be writing to you :heart: 
Your code is running on my machine, and the code you wrote works properly :+1: 
However, that's all the good things I can say about your project.
You over-relied on divs, making the flow of your code almost impossible to follow, and there is no use at all of the semantic tags. :crying_cat_face: 
I think it would be better to use semantic tags instead of divs as much as possible.
Furthermore, your code documentation is doodoo :hankey: 
I think it would be better for you to use the documentation template provided by microverse, It's a great starting point. :fire:

**Review by de-scholar — CHANGES_REQUESTED** [review]
Hi @AlexHamn , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._
- [ ] Great review summary, but I would like to suggest you try to use inline comments so that it will be very easy for the student to know which code to change exactly.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section id="welcome-section">
+        <h1>Hey I am Mimic</h1>
+        <br />
+        <p>a web developer</p>
+      </section>
+      <div class="style">
```
Comment (AlexHamn) [inline index.html:25]: You may want to use a more descriptive class in order to give context to a div.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        <div id="projects">
```
Comment (AlexHamn) [inline index.html:26]: This coud be a <section> instead of a <div>

**Review by smcommits — APPROVED** [review]
### Status: Approved 👍🏽 

Hi @AlexHamn, 

## You have done a great job! Your project is complete. This PR is now approved. You can go ahead and merge the branch. Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**